docs/photoniq/vwrs/03-configuration-and-setup/01-configuring-edgeworkers.md

```diff
@@ -0,0 +1,126 @@
+---
+sidebar_position: 30
+title: Configure a VWRs EdgeWorker
+---
+
+# Overview
+
+VWRs require [Akamai EdgeWorker Documentation](https://techdocs.akamai.com/edgeworkers/docs) to execute functions at the edge to help redirect and optimize web traffic for your websites. To get started using EdgeWorkers with VWRs, you need to import the `VirtualWaitingRoom` class from the photoniq-vwrs-client using Node Package Manager (NPM). This integrates the VWRs with the EdgeWorker. 
+
+## Setting Up EdgeWorkers
+
+### Prerequisites
+To configure EdgeWorkers for use with VWRs, you need the following:
+
+- An [Akamai EdgeWorker resource tier](https://techdocs.akamai.com/edgeworkers/docs/resource-tier-limitations): Because the VWRs EdgeWorker performs up to two sub-requests, this resource tier must allow at least two HTTP sub-requests on the `onClientRequest` event handler. 
+- [Akamai Property Manager user-defined variables](https://techdocs.akamai.com/edgeworkers/docs/request-object#setvariable): This helps set the value of the Property Manager user-defined variable to control how Akamai controls network traffic. PhotonIQ uses about 350 characters out of the 1024 limit set by Akamai. 
+
+### Step 1: Install the PhotonIQ VWRs Client
+
+Before you can set up an EdgeWorker, you must install `@macrometa/photoniq-vwrs-client`.
```

Comment:
I think Naimish said that this is not what we support anymore.

Reply:
We are going to need @naimishvora-macrometa to help us come up with the correct steps here.